pkg/lib/facade/coordinator.go

```diff
@@ -1425,3 +1427,17 @@ func (c *Coordinator) GetUsersByStandardAttribute(attributeName string, attribut
 
 	return uniqueUserIDs, nil
 }
+
+func (c *Coordinator) GetUserByLoginID(loginIDKey string, loginIDValue string) (string, error) {
+	info, err := c.Identities.AdminAPIGetByLoginIDKeyAndLoginIDValue(loginIDKey, loginIDValue)
+
+	if errors.Is(err, api.ErrIdentityNotFound) {
+		return "", api.ErrUserNotFound
+	} else if errors.Is(err, loginid.ErrNormalize) {
```

Comment:
Since ErrNormalize is removed, and the error is constructed downstream, we do not need this if statement checking.

Reply:
Removed `ErrNormalize` related code